ITISFoundation/osparc-simcore
# 🐛storage: fixes query to file metadata

## What do these changes do?

- 🐛 Fixes bug introduced with [this change](https://github.com/ITISFoundation/osparc-simcore/pull/5796/files#diff-0e92b3dc5f2ea70b28592e67fd63e75c9767cfb1c3863d12b2a8956f8d8d2366) in the database query
- 🐛 Fixes access rights to storage data (in coordination with @matusdrobuliak66 )

## Related issue/s

- resolves ITISFoundation/osparc-issues#1587

## How to test

- The changes were **driven** by to **tests** in `storage`, namely
   - Test to check correctness of sql expression (also to print an manually test against master db)
       -  `pytest -vv tests/unit -k test_building_sql_statements`
   - Test to checks different queries in `GET locations/{loc_id}/files/metadata`
       - `pytest -vv tests/unit -k test_get_files_metadata`
- **Manual exploratory testing** using steps in  ITISFoundation/osparc-issues#1587 on a local deploy

## Dev-ops checklist
